example/react/ProductBookmark.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { FC, MouseEvent, useState } from 'react'
+import { defineMessages, useIntl } from 'react-intl'
+import { IconBookmark } from 'vtex.store-icons'
+import { withToast } from 'vtex.styleguide'
+import { useCssHandles } from 'vtex.css-handles'
+import useProduct from 'vtex.product-context/useProduct'
+
+const CSS_HANDLES = ['productBookmark'] as const
+
+const messages = defineMessages({
+  added: {
+    id: 'store/wishlist.added',
+    defaultMessage: '',
+  },
+  removed: {
+    id: 'store/wishlist.removed',
+    defaultMessage: '',
+  },
+})
+
+const Bookmark: FC<Props> = ({ showToast }) => {
+  const handles = useCssHandles(CSS_HANDLES)
+  const { formatMessage } = useIntl()
+  const [filled, setFill] = useState(false)
+  const productContext = useProduct()
+
+  const handleClick = (ev: MouseEvent) => {
+    ev.stopPropagation()
+    ev.preventDefault()
+    setFill(!filled)
+    if (productContext && productContext.product) {
+      // eslint-disable-next-line no-console
```

Comment:
Is this `console.log` supposed to be here?

Reply:
yess, as an example of how to use data from context